kolibri/plugins/learn/assets/src/views/LibraryPage/index.vue

```diff
@@ -77,7 +81,7 @@
           />
           <!-- Other Libraries -->
           <OtherLibraries
-            v-if="showOtherLibraries"
+            v-if="showOtherLibraries && !isLearner"
```

Comment:
Not sure why we're now hiding the other libraries from learners?

This basically creates no change from the previous experience for learners, even though there are browsable libraries on the network:

![image](https://github.com/user-attachments/assets/c38bb9cf-1fa2-4852-a94f-051cab878b50)